app/AppContainer.tsx

```diff
@@ -59,7 +60,7 @@ const App = memo(({ root, isMasterDetail }: { root: string; isMasterDetail: bool
 				}
 				Navigation.routeNameRef.current = currentRouteName;
 			}}>
-			<Stack.Navigator screenOptions={{ headerShown: false, animation: 'none' }}>
+			<Stack.Navigator screenOptions={{ headerShown: false, animation: 'none', navigationBarColor: themes[theme].surfaceLight }}>
```

Comment:
It seems to work, but introduces a regression on Android:
- Open gallery and choose a photo
- Share to our app
- Select any room
- Composer is rendered

Make sure to reload metro and go through all the steps after a change, since fast refresh doesn't work.